content/docs/installation-try-react.md

```diff
@@ -0,0 +1,21 @@
+---
+id: installation-try-react
+title: Try React
+permalink: docs/installation-try-react.html
+redirect_from:
+  - "docs/installation.html"
+  - "download.html"
+  - "downloads.html"
+  - "docs/tooling-integration.html"
+  - "docs/package-management.html"
+  - "docs/language-tooling.html"
+  - "docs/environments.html"
```

Comment:
The new Installation pages should have prev/next frontmatter links 😄

Reply:
See #451 - the lack of sync between the titles and the ids causes the next/prev to appear as "Installation New Apps", etc

Reply:
Not sure I understand. The next/prev titles are currently generated based on the URL you put in the next/prev frontmatter (so it's something we control, even if it's not synced).

I think the new section should still have next/prev links to feel more consistent with the others.

Reply:
```
//  installation-existing-app.md
id: installation-existing-app
title: Add React to an Existing Application
```

```
// elsewhere
next: installation-existing-app.html
```

Results in:

![image](https://user-images.githubusercontent.com/1571667/34284507-01889780-e67f-11e7-8dde-18819918e54c.png)

Instead of:

![image](https://user-images.githubusercontent.com/1571667/34284525-220d518a-e67f-11e7-847d-77cbf02e16b6.png)

This is because of #451 where the url is parsed instead of looking up the title in the TOC data structure. I am planning on fixing that and adding next/prev once it is done, but **I can certainly adjust the filenames for now so that it looks correct**.

Reply:
> so it's something we control, even if it's not synced

You chose to name the new file `installation-existing-app.md`. So I'm just saying, before #451 is resolved, we could work around this issue by just naming it `add-react-to-an-existing-app.md` instead. 😄